src/main/java/uk/gov/hmcts/reform/hmc/controllers/PartiesNotifiedController.java

```diff
@@ -40,8 +42,11 @@ public PartiesNotifiedController(PartiesNotifiedService partiesNotifiedService)
     })
     public void putPartiesNotified(@RequestBody @Valid PartiesNotified partiesNotified,
                                    @PathVariable("id") Long hearingId,
-                                   @RequestParam("version") int responseVersion) {
-        partiesNotifiedService.getPartiesNotified(hearingId, responseVersion, partiesNotified);
+                                   @RequestParam("version") int requestVersion,
+                                   @RequestParam("received")
+                                   @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ss")
```

Comment:
If you haven't already, could you please double-check this `pattern` works with the values that we send in our responses and place on the topic please for the `receivedDateTime`? I suspect it might not, if this is forcing the format to have a space between the date and time.

We should expect consumers to be sending us values in exactly the same format as we provide to them.

Reply:
Sorry, the pattern came from another class (UnNotifiedHearingsController) but has now been corrected.